gravity.sh

```diff
@@ -858,10 +899,30 @@ Options:
   exit 0
 }
 
+repairSelector() {
+  case "$1" in
+    "recover") recover_database=true;;
+    "recreate") recreate_database=true;;
+    *) echo "Usage: pihole -g -r {recover,recreate}
+Attempt to repair gravity database
+
+Available options:
+  pihole -g -r recover    Try to recover a damaged gravity database file.
```

Comment:
```suggestion
  pihole -g -r recover [force]    Try to recover a damaged gravity database file.
```

Reply:
I did not document the `force` option at all as I don't think there can be any type of corruption that is not detected by either the integrity or the foreign key checks. It just be there as a last-resort option that may be worth trying when everything else failed. Recovery is a fragile task consuming a lot of resources and shouldn't be done if not necessary.

Reply:
My only concern is that even we will forget about the option if it is not documented. We could expand the explanation by your last sentence to make it clear that this not something users should do when there is no problem.

```
If "force"is set, Pi-hole will run the recovery process even when no damage is detected. This option is meant to be a last resort. Recovery is a fragile task consuming a lot of resources and shouldn't be done if not necessary.
```

Reply:
@DL6ER, any final response here?

Reply:
I chose another solution, please check if you agree with this.